content/docs/next/triage/issues.md

```diff
@@ -1,7 +1,5 @@
 ---
-title: Issue Triage Guidelines
-weight: 8100
-description: Managing incoming issues
```

Comment:
The description doesn't add value -- the title is clear enough -- so I dropped it.